### ISSUE TITLE
parse_robot creates a new section as wrongly.

### ISSUE THREAD

**Issue by dnzguzel** [gurock/trcli#167]
### TestRail CLI Version

1.6.1

### CLI Environment

MacOS, Python 3.10.4

### TestRail Version

7.0.2.1016

### TestRail Instance Type

Professional Server

### Current behavior

After pushing the Robot Framework results to Testrail, a new section is created under the test cases sections. The section name is the same as the Automation ID. For example, like "TestCases.Mobile.Mbl 2 Main Page Tests". It just does this with parse_robot. This does not happen with parse_junit.

**Command**:

```
trcli -y \
-h https://testrail.company.com/ \
--project ProjectName \
--username $TESTRAIL_USER \
--password $TESTRAIL_PASS \
parse_robot \
--title RF_Automated_Local_Test \
-f output.xml

```
**Output**:
![image](https://github.com/gurock/trcli/assets/29749553/8a13bac0-45d2-4f84-b28e-ea58adaade39)

### Desired behavior

I desire that it should not create a new section. Because I already created a section.

**Comment by bitcoder** [comment]
Hi  @dnzguzel ,

1) not sure why you mentioned "parse_junit". Can you please clarify?
2) you mentioned "because I already created a section".. how exactly? manually? Or as a consequence of a previous import of test reusults?
3) from my understanding, you imported the Robot Framework results and it created that section you mentioned. Then you imported again the RF results and it created a new section. Is that it? If not, can you please clarify the exact steps you did and  exactly at which one the behaviour you got was not what you were expected?

Thanks

**Comment by dnzguzel** [comment]
Hi @bitcoder ,
Before using parse_robot, I was using parse_junit and sending my robot test results to the test runs of the test scenarios under the folders I added. However, with parse_robot, my test results still go to the test runs of the relevant scenarios, but empty folders are created as in the screenshot. This does not happen when using parse_junit. It only happens with parse_robot.

**Comment by bitcoder** [comment]
Can you please an example of the output.xml you're trying to upload, to replicate this?

**Comment by dnzguzel** [comment]
Solved with 1.6.3 version.
